From: Jeremy Bicha <jbicha@debian.org>
Date: Fri, 11 Mar 2022 09:06:40 -0500
Subject: Revert "sharing: Don't manage gnome-remote-desktop"

This reverts commit 58add5197e6b3552502d6b30de0c9eadf9c92014.

We need gnome-control-center 42 to land in Debian first or
at least simultaneously for this commit to be useful.
---
 plugins/sharing/gsd-sharing-manager.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/sharing/gsd-sharing-manager.c b/plugins/sharing/gsd-sharing-manager.c
index aca78b3..43067cc 100644
--- a/plugins/sharing/gsd-sharing-manager.c
+++ b/plugins/sharing/gsd-sharing-manager.c
@@ -98,6 +98,7 @@ static gpointer manager_object = NULL;
 
 static const char * const services[] = {
         "rygel",
+        "gnome-remote-desktop",
         "gnome-user-share-webdav"
 };
 
